recipe/build.sh

```diff
@@ -1,32 +1,22 @@
 #!/bin/bash
-set -ex
 
-osname=`uname`
+meson builddir/ \
+  -Dbuildtype=release \
+  -Dprefix=$PREFIX \
+  -Dlibdir=lib \
+  -Dplatforms=x11 \
+  -Dosmesa=gallium \
+  -Dosmesa-bits=8 \
+  -Dvulkan-drivers=[] \
+  -Dgallium=true \
```

Comment:
I think this might work:

```suggestion
  -Dosmesa=true \
  -Dosmesa-bits=8 \
  -Dvulkan-drivers=[] \
```